vdev/src/testing/integration.rs

```diff
@@ -28,11 +29,28 @@ impl IntegrationTest {
         let integration = integration.into();
         let environment = environment.into();
         let (test_dir, config) = IntegrationTestConfig::load(&integration)?;
+        // Load thee compose_config
```

Comment:
```suggestion
        // Load the compose_config
```

Reply:
Though the comment does feel a little self-evident given the function it's describing, at least to me.